src/common/modules/Logger.js

```diff
@@ -59,6 +91,8 @@ export function log(...args) {
 export function logError(...args) {
     args.unshift(MESSAGE_LEVEL.ERROR);
 
+    args = verifyArgs(args);
```

Comment:
It's enough if you do it once. All these `logXY` also call `log` at the end, so you actually run the args through the whole `verifyArgs` twice. (same below)